mlir/test/Dialect/XeGPU/invalid.mlir

```diff
@@ -132,18 +132,10 @@ func.func @subgroup_load_nd_9(%src: memref<4x8x16xf16>) {
   return
 }
 
-// -----
-func.func @subgroup_load_nd_offset_1(%src: memref<4x8x16xf16>, %x : index) {
-  %1 = xegpu.create_nd_tdesc %src: memref<4x8x16xf16> -> !xegpu.tensor_desc<16xf16>
-// expected-error@+1 {{Mismatched ranks between offsets and tensor descriptor}}
-  %2 = xegpu.load_nd %1[0, 0] : !xegpu.tensor_desc<16xf16> -> vector<16xf16>
-  return
-}
-
```

Comment:
[decided that this is a valid case now](https://github.com/llvm/llvm-project/pull/162095#discussion_r2446845798)